src/Umbraco.Web.UI.Client/src/views/components/contenttype/umb-content-type-property.html

```diff
@@ -100,8 +100,10 @@
 
           <div class="umb-group-builder__property-tag" ng-if="vm.property.inherited">
               <umb-icon icon="icon-merge"></umb-icon>
-              <span style="margin-right: 3px"><localize key="contentTypeEditor_inheritedFrom">Inherited from</localize></span>
+            <span style="margin-right: 3px"><localize key="contentTypeEditor_inheritedFrom">Inherited from</localize></span>
+            <a ng-click="vm.clickComposition(vm.property.contentTypeId)">
               {{vm.property.contentTypeName}}
+            </a>
```

Comment:
@patrickdemooij9 I think this should use `<button type="button">` from an web accessibility point of view unless it linked to something. It could use various classes like `btn-link`.

For example:

```
<button type="button" class="btn btn-small btn-link" ng-click="vm.clickComposition(vm.property.contentTypeId)">
    {{vm.property.contentTypeName}}
</button>
```

Reply:
Thank you for the feedback! I've updated it to use a button while still making sure that the styling is the same